Formula/kustomize.rb

```diff
@@ -3,11 +3,11 @@ class Kustomize < Formula
   homepage "https://github.com/kubernetes-sigs/kustomize"
   version "4.2.0"
   on_arm do
-    url "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.2.0/kustomize_v4.2.0_darwin_arm64.tar.gz"
+    url "https://github.com/kubernetes-sigs/kustomize/releases/download/kustomize%2Fv4.5.7/kustomize_v4.5.7_darwin_arm64.tar.gz"
```

Comment:
Per [this version chart](https://github.com/kubernetes-sigs/kustomize#kubectl-integration).